### DISCUSSION TITLE
Add additonal two players

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [Wohlstand/TheXTech#693] [Ideas]
It would be nice if the x tech supported up to four players local / online mixed.
Possibly up to 256 for the editor only.

**Comment by ds-sloth** [comment]
This is planned, but is a very large feature so I'm converting this into a discussion for now. NetPlay is not available yet, but is planned.

Local >2P is currently in-progress and you can access an **experimental** version of it using the cheat `4shared` on the main branch for 4P shared-screen, or `4split` on the multires branch for 4P splitscreen. `4split` should be used together with another cheat, `customview`, to get a >1280x720p screen.

I'm really glad you're interested in this! Please let me know here or create issues if anything comes up during your testing.

**Reply by Trickiy** [reply]
>NetPlay is not available yet, but is planned.

Would downloading maps from a level viewer kinda like course bot in Mario maker 2 be added?

**Reply by ds-sloth** [reply]
Maybe, but it's not necessary for an engine like TheXTech that supports file-based downloads of content from your browser.

**Reply by Superbloxen** [reply]
I remember Wohlstand mentioning a potential system where episodes could be downloaded from the SMBX Preservation Archive in TheXTech in an issue once..

**Reply by Trickiy** [reply]
>but it's not necessary for an engine like TheXTech that supports file-based downloads of content from your browser.

But the old browsers like the 3ds and Wii browsers are not supported. Also I and many others don't want to have to take out there sd card and put it back in; every time that they want to play a new episode or use a new asset pack. If thextech was only pc compatible it would truly be unnecessary.
